free-2.md

```diff
@@ -46,6 +89,10 @@ aren't and conjecture why not. If so, describe them and provide links. How well
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+There are lots of other DSLs in this domain. Google Inbox, for example, tries
+to combine email and todos, but doesn't have great support for mandatory
+events. I got the idea of importing from email from Google as well, but their
+system doesn't work as well as I hoped.
```

Comment:
A lot of task-trackers and ticketing systems too. JIRA, Azendoo, Phabricator. And then Todoist, and things like it. Google Calendar is also highly related, but obviously more limited.